.github/workflows/gh-pages.yml

```diff
@@ -74,8 +74,10 @@ jobs:
 
       - name: Run benchmarks
         run: |
-          for bench in call dict gil list pyclass pyobject set tuple; do
-            cargo bench --features hashbrown --bench "bench_$bench" -- --output-format bencher | tee -a output.txt
+          python -m pip install --upgrade pip && pip install nox
```

Comment:
This installs Nox but does not seem to use it?

Reply:
Yeah it should be `nox -s bench` in the command below, will fixup 🤦